### PR TITLE
created endpoint for create Submission

### PR DESCRIPTION
**Pull request by d-username**
--> created endpoint for creating Submission
--> added a little update to Schema - "UpdatedAt"
--> tested with Insomnia

### CODE REVIEW

**Comment by d-username** [comment]
#160    Issue number

File: prisma/schema.prisma [hunk 1]
```diff
@@ -188,5 +188,5 @@ model Submission {
   userId           Int
   user             User           @relation(fields: [userId], references: [id])
   createdAt        DateTime       @default(now())
-  updatedAt        DateTime
+  updatedAt        DateTime       @updatedAt
```
Comment (dearshrewdwit) [inline prisma/schema.prisma:191]: Nice catch.

File: src/controllers/user.js [hunk 2]
```diff
@@ -218,3 +219,28 @@ export const updateNoteById = async (req, res) => {
...
+export const submitExercise = async (req, res) => {
+  const cohortExerciseId = Number(req.params.id)
+
+  const { userId } = req.body
+
+  const newSubmissionData = {
+    userId,
+    cohortExerciseId
+  }
+  console.log(newSubmissionData)
```
Comment (dearshrewdwit) [inline src/controllers/user.js:232]: Remove debugging statement

File: src/controllers/user.js [hunk 3]
```diff
@@ -218,3 +219,28 @@ export const updateNoteById = async (req, res) => {
     return sendDataResponse(res, 400, { err: err.message })
   }
 }
+
+export const submitExercise = async (req, res) => {
```
Comment (dearshrewdwit) [inline src/controllers/user.js:223]: Rename method to `createSubmission`

**Review by dearshrewdwit — CHANGES_REQUESTED** [review]
Nice.

**Review by dearshrewdwit — APPROVED** [review]

**Pull request merged**